### DISCUSSION TITLE
I've been approved for the student developer pack, but my status does not change to 'Pro'

### DISCUSSION THREAD

**Discussion by a2jinhee** [community/community#109530] [GitHub Education]
### Hi everyone,

Hello, I've been waiting for about three days for the 'Pro' status!
I've been approved for the student developer pack, but my status does not change to 'Pro'.

**Comment by aradiyo** [comment]
me too

**Comment by namith1003** [comment]
![image](https://github.com/community/community/assets/65733592/61c3f0f3-5ba1-42c9-9d39-2b202b6b2a73)

I think even though it says approved, the other pages state that the application is still in processing and the features haven't yet been handed, check if yours is the same as mine here and if not im not fully sure, but if so you may need to stay for a few more days thats all. 

Hopefully this helps answer your question.

**Comment by Minakshig154** [comment]
No luck still same problem for me any one from GitHub care to help

**Comment by Brijesh1963** [comment]
My GitHub developer pack was approved but till now ,on the application section not able to apply for campus expert...
Please look at this matter, last date of apply 29th Feb.

**Comment by a2jinhee** [comment]
Hello! First, thank you all for your interest in this matter. 
In my case, I applied for the student development pack and was approved on 2/21. But my status was converted to a 'pro' after a few days (2/24). I think it takes some time after the approval to actually get the full benefits of the student development pack. I'm closing as this is resolved!
